docs/source/index.rst

```diff
@@ -46,3 +44,20 @@ Welcome to HackRF's documentation!
    expansion_interface
    hardware_triggering
    rf_shield_installation
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Opera Cake Documentation
+
+   opera_cake
+   opera_cake_faq
+   opera_cake_hardware
+   opera_cake_board_addressing
+   opera_cake_port_configuration
+   opera_cake_modes_of_operation
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Jawbreaker Documentation
+
+   jawbreaker      
```

Comment:
I don't like moving the Jawbreaker page away from the HackRF One page unless you plan to also move HackRF One to its own section. At a later time I think we should add a page for rad1o which is the third HackRF platform supported by the software in this repo. I would like either each platform to have its own section or all three platforms to be in the same section.

I notice that the "Hardware" section is almost entirely about HackRF One and could be converted to a HackRF One section. If you are okay with doing that in the future, then I am okay with moving Jawbreaker here.